release-notes.gemspec

```diff
@@ -10,7 +9,7 @@ Gem::Specification.new do |spec|
   spec.email         = ["dvmonroe6@gmail.com"]
 
   spec.summary       = "Release notes for stakeholders"
-  spec.description   = spec.summary
+  spec.description   = "A tool that generates release notes for your project."
```

Comment:
Maybe summary is no longer relevant either if it needs to be different? Perhaps we also need to update the README since with some of your changes - it's not just for stakeholders...